### ISSUE TITLE
Drop Lowest Score for teams

### ISSUE THREAD

**Issue by Allicolyer** [Iridescent-CM/technovation-app#1537]
If all teams get 5 scores, we would like to drop the lowest score when calculating the average

**Comment by joemsak** [comment]
to QA: be sure you have at least one submission with 5 complete semifinals scores
- Keep track of the average semifinals score of that submission
- Keep track of which score is the lowest one

- @Allicolyer you will have to run this script on the terminal:
`heroku run rails drop_lowest_scores -a technovation-qa`

Verify that your submission has 4 (or one less) scores
Verify that the average semifinals score went up
Verify that the previously lowest score is now gone

**Comment by joemsak** [comment]
@Allicolyer @amyiridescent @madhavitechnovation Do you want it so teams have 4 scores at most, or to only drop the lowest score if they have 5 or more?

EXAMPLE

Team A has 4 scores - nothing is dropped
Team B has 5 scores - 1 score is dropped
Team C has 6 scores - 2 scores are dropped

OR do you want it like this

Team A has 4 scores - nothing is dropped
Team B has 5 scores - 1 score is dropped
Team C has 6 scores - 1 score is dropped

**Comment by Allicolyer** [comment]
Thanks for working so quick on this@joemsak. Let's please hold on this. We need to make a decision on the program side!

**Comment by Allicolyer** [comment]
@joemsak we want all teams to have a score dropped. With a couple of exceptions.

**Comment by Allicolyer** [comment]
@joemsak how can we make sure that the judges still get the proper certificates, even if their scores are deleted?

**Comment by joemsak** [comment]
Isn't that what #1535 is for?

Joe Sak
Technovation Web Development

SMS
+1 (312) 945-6317

**Comment by Allicolyer** [comment]
not quite, that is for judges who email us and need a different certificate.

This ticket will lead us to drop around 240 scores and we can't keep track of which scores got dropped and who judges are and if they are being demoted by us dropping their scores.

**Comment by joemsak** [comment]
I think what you're asking for is a very high risk of unnecessary
complexity for the software, if you insist on dropping scores, and
finalists are already being chosen and announced anyway

Joe Sak
Technovation Web Development

SMS
+1 (312) 945-6317
